bigflow/version.py

```diff
@@ -1,46 +1,64 @@
 import re
+from typing import Optional
+from uuid import uuid1
```

Comment:
why uuid1?

Reply:
https://docs.python.org/3.7/library/uuid.html#uuid.uuid1
looks like the best option